GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/MoveRenameFolderTests.cs

```diff
@@ -128,7 +128,7 @@ public void MoveFullFolderToFullFolderInDotGitFolder()
         {
             string fileContents = "Test contents for MoveFullFolderToFullFolderInDotGitFolder";
             string testFileName = "MoveFullFolderToFullFolderInDotGitFolder.txt";
-            string oldFolderPath = this.Enlistment.GetVirtualPathTo("MoveFullFolderToFullFolderInDotGitFolder");
+            string oldFolderPath = this.Enlistment.GetVirtualPathTo("MoveSourceFolder");

```

Comment:
Is there some documentation or something we can put in a comment here so it doesn't get changed back at some point?